### PR TITLE
Create & View Articles from db

### PR DESCRIPTION
**Pull request by elvisiraguha**
# Description

This PR contains changes set up the routes for creating and fetching articles

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `ft-view-articles-db`
- Run the command `yarn install`
- Run the command `yarn start` to run the server locally

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)
Create an article
shot-2020-08-12_06-33-05
View individual article
shot-2020-08-12_06-34-12
View all articles
shot-2020-08-12_06-34-50

### CODE REVIEW

File: Server/controllers/articles.controller.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Article from "../models/Article.model.js";
+import Response from "../helpers/responses.js";
+
+class Articles {
+  static get = async (req, res) => {
+    Article.find()
+      .then((articles) => {
+        Response.success(res, 200, "Articles fetched successfully", articles);
+      })
+      .catch((err) => Response.error(res, 404, "Articles Not Found"));
```
Comment (bdushimi) [inline Server/controllers/articles.controller.js]: @elvisiraguha This is the server error (server encountered an error and hence couldn't execute the request), 5XX HTTP code is to be used, not 4XX HTTP code. Kindly verify and update this elsewhere.

**Review by bdushimi — CHANGES_REQUESTED** [review]

**Comment by bdushimi** [comment]
@elvisiraguha test coverage has dropped significantly! Do you know why?

**Comment by elvisiraguha** [comment]
> @elvisiraguha test coverage has dropped significantly! Do you know why?

There is a test I skipped, I am still doing research on how to use mock data when testing.

File: Server/controllers/articles.controller.js (hunk 1 above)
Comment (elvisiraguha) [reply Server/controllers/articles.controller.js]: Updated.

**Pull request merged**